packages/jest-types/src/Config.ts

```diff
@@ -494,6 +494,11 @@ export type ProjectConfig = {
   workerIdleMemoryLimit?: number;
 };
 
+export type ProjectConfigPerfStats = {
```

Comment:
I don't think this belongs in `Config`. Seems more like a `TestResult` thing

Reply:
Ah yea, good call!

Considering this just captures `setupAfterEnv` `perfStats` what are your thoughts on `SetupAfterEnvPerfStats`? I think the reason I probably avoided using `TestResult` here was because this is only a subset of the `TestResult` `perfStats` due to where I can capture this metadata during the lifecycle of test execution.